engine/liveupdate/src/liveupdate.cpp

```diff
@@ -391,7 +391,7 @@ namespace dmLiveUpdate
             dmLogWarning("Failed to set manifest to mounted archive uri: %s:%s/%s\n", uri.m_Scheme, uri.m_Location, uri.m_Path);
             return dmResource::RESULT_INVALID_DATA;
         }
-
```

Comment:
`dmResourceProvider::SetManifest()` removes the old one, that means we need to update pointer to the new manifest in `g_LiveUpdate`